massgov/mayflower
# DP-15965 Change directions to be capitalized

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description
I changed directions to be capitalized in the contact-item pattern. Also I adjusted the contact-group json to show this in the contact-group pattern. Finally I updated a backstopJS reference screenshot after this change.

## Related Issue / Ticket

- https://jira.mass.gov/browse/DP-15965

## Steps to Test


1. You will see on the live site that "directions" is not capitalized on the locations page: 
https://www.mass.gov/locations/self-service-dta-kiosk-at-bristol-community-college-taunton-center-limited-services

2. Now with this change you will see on the location page that directions is capitalized: 
http://localhost:3000/?p=pages-location-general-content

3. Also I adjusted the contact-item group json to show this:
http://localhost:3000/?p=molecules-contact-group-as-address

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
